### PR TITLE
Info

### PR DESCRIPTION
**Pull request by DivyanshSareen**
Added prismjs to all files, removed carbon iframes. Added a temporary list for scrollspy

Note: image component is not displaying images right now, it might work after merge

### CODE REVIEW

File: public/card.html [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+            <i class="fa-solid fa-xmark"></i>
+        </div>
+    </div>
+    <div class="card-img resp-img">
+        <img src="../resources/red-panda.jpeg" alt="card-image">
+    </div>
+    <div class="card-badge">
+        <div class="badge">new</div>
+        <div class="badge">sale</div>
+        <div class="badge">somethin</div>
```
Comment (zoyatanweer) [inline public/card.html]: should be "something" instead of somethin

**Review by zoyatanweer — APPROVED** [review]
looks good to merge. Code is easy to understand.

**Comment by zoyatanweer** [comment]
have you provided correct path to your images? check once again.

**Review by zoef-yt — APPROVED** [review]
Suggestion changes the sidebars to be sticky. The code looks good. Good to merge.

**Comment by DivyanshSareen** [comment]
> have you provided correct path to your images? check once again.

yeah, it is working locally maybe the images need to be merged in the dev branch

**Pull request merged**